src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -426,16 +426,38 @@ has to be updated, then for MEG it could be considered to be a new session.
 
 ## Empty-room MEG recordings
 
-Empty-room MEG recordings capture the environment and system noise. Their
-collection is RECOMMENDED, before/during/after each session. This data is stored
-inside a subject folder named `sub-emptyroom`. The `session label` SHOULD be
-that of the date of the empty-room recording (e.g. `ses-YYYYMMDD`). The
-`scans.tsv` file containing the date/time of the acquisition SHOULD also be
-included. Hence, users will be able to retrieve the empty-room recording that
-best matches a particular session with a participant, based on date/time of
-recording.
+Empty-room MEG recordings capture the environmental and recording system's
+noise.
+There are two widely used strategies for collecting these recordings:
 
-Example:
+1.  Some labs record one empty-room measurement per day, which is then shared by
+    all researchers that record subject data on that day
+
+1.  Some labs make it the responsibility of each researcher to record a short
+    empty-room measurement prior to each subject-specific recording
+    (e.g., while preparing the study subject for the task)
+
+For cases resembling the **first** strategy, it is RECOMMENDED to store the
+empty-room recording inside a subject folder named `sub-emptyroom`.
+If a `session-<label>` entity is present, its label SHOULD be the date of the
+empty-room recording in the format `YYYYMMDD`, i.e., `ses-YYYYMMDD`.
+For these cases, the `scans.tsv` file containing the date and time of the
+acquisition SHOULD also be included.
+
+However, for cases resembling the **second** strategy, the subject data and
+empty-room recording are naturally linked and it is thus RECOMMENDED to instead
+store the empty-room recording inside the study subject's folder.
+
+Furthermore, in **both** cases the label for the `task-<label>` entity in the
+`*_meg.json` file SHOULD be set to `noise`.
+
+The rationale is that this naming scheme will allow users to easily retrieve the
+empty-room recording that best matches a particular experimental session, based
+on date and time of the recording.
+In the context of BIDS it is RECOMMENDED to perform an empty-room recording for
```

Comment:
does this mean you are recommending strategy 2?

none of the existing BIDS strategies are following strategy 2. They all are formatted according to strategy 1

Reply:
You mean "none of the existing BIDS datasets that are shared and that you are aware of". 

from Gross' et al. [Good practice for conducting and reporting MEG research](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3925794/) comes this recommendation: 

> Before the participant enters the MEG laboratory: We recommend that the condition of the MEG system is recorded by performing an “empty room” recording of about 2 minute duration before and after the experiment.  

Although in the Good practice recommendations we subsequently recommend storing it in a longitudinal database to identify technical malfunctions, which is more according to strategy 1 and which would lead to multiple noise recordings per day. The relevance of noise measurements during data analysis depends on the analysis methods and experimental design and hence is study-specific.

Reply:
I mean if we are recommending strategy 2 as preferred using the keyword RECOMMENDED, the "example datasets" (ds000117 and others) should also adhere to this recommendation? Otherwise, it would seem that we cannot follow our own recommendations ...

Reply:
IMHO the BIDS project (i.e. "we" in this context) is not to recommend any scientific research approaches, strategies on how to record or strategies how to analyze. BIDS is only about strategies for organizing data and metadata. Strategies on how to do scientific work should be left to the respective domain-specific experts, as e.g. Gross et al. or OHBM COBIDAS, or other authors/groups that work on and publish recommendations. BIDS should accommodate scientific practices recommended elsewhere.

Reply:
absolutely agree. But in my understanding researchers can continue to record data according to the two strategies but that doesn't necessarily need to translate into different organization schemes for the two strategies.

Reply:
re: formatting our example datasets according to our own recommendations:

> Otherwise, it would seem that we cannot follow our own recommendations

yes, I agree that it'd be good to go over the examples and update them ... because as the standard gets more refined, the datasets should keep up. I am constantly doing that with my own example [`eeg_matchingpennies`](https://github.com/bids-standard/bids-examples/tree/master/eeg_matchingpennies) but apparently it is too much to expect other example contributors to do the same (I say this in a matter-of-fact, neutral voice).

Placing this burden on the maintainers is really a lot to ask: we are few and there is LOTS to do. We somehow need to figure this out (should be its own discussion though IMO)

Reply:
> researchers can continue to record data according to the two strategies but that doesn't necessarily need to translate into different organization schemes for the two strategies.

that is true. My thoughts:

- we cannot elevate the two RECOMMENDATIONS how to organize noise data to a single REQUIRED way how to organize nose data --> it's not backwards compatible
- we *could* turn the two recommendations into a *single* recommendation (need to discuss which one makes more sense)

Yet, software would still need to be able to deal with both ways.

Hence, I think the PR in its current state is "okay".

- it is clarifying the current state, adding examples and missing representations
- it is not making it easier for software devs unfortunately

Reply:
> I know, but that's exactly my point. Don't change things just because it seems nicer. It should be driven by actual use cases from users

I absolutely agree with you and I don't do these PRs to torture myself and my reviewers :laughing: 

This whole PR *was* driven by @hoechenberger reading that part of the spec and being confused: https://github.com/mne-tools/mne-bids/pull/421#issuecomment-633005511

and from there it went on like it often does:
- a simple clarification
- noticing that it's actually much much complicated
- going way beyond the original intentions, trying to solve lots of issues at once because it's all interrelated 
- the whole thing blows up

From what I have seen in the last 2 years, this process then usually ends in one of three ways:

1. the PR goes stale, because there are a lot of changes and it's too much to handle, taking too long, and people are losing interest
1. the PR is merged and it's an improvement and everybody is happy
1. the PR is merged, but it's not very clear whether everybody is happy

I am not sure how we should proceed with _this_ PR.

Reply:
This is a very good summary of how the process often goes and I think it deserves to be placed somewhere more visible as reminder for future issues like this.

Reply:
> I know, but that's exactly my point. Don't change things just because it seems nicer. It should be driven by actual use cases from users
>> This whole PR was driven by @hoechenberger reading that part of the spec and being confused: mne-tools/mne-bids#421 (comment)

right, but this was a good opportunity to direct @hoechenberger to mess with the validator code (cc @agramfort ), add a simple change to raise a warning if `sub-emptyroom` did not have `task-noise`. I wrote the MEG part of the validator and I'm aware it is deficient in many respects because we had to push out the paper. You can write whatever you want in the specification but ultimately people will follow the validator. If it passes, then upload to openneuro.